Tests/Shared.Specs/Shared.Specs.projitems

```diff
@@ -9,81 +9,79 @@
-    <Compile Include="$(MSBuildThisFileDirectory)NullableNumericAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)NullableSimpleTimeSpanAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)NumericAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ObjectAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ObjectCastingSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ObjectExtensionsSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)PropertyInfoAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)PropertyInfoSelectorAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)PropertyInfoSelectorSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ReferenceTypeAssertionsSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)SimpleTimeSpanAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)StringAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TaskAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TaskOfTAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Numeric\ComparableSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Numeric\NullableNumericAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Numeric\NumericAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\BooleanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\DateTimeAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\DateTimeOffsetAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\DateTimeOffsetValueFormatterSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\EnumAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\GuidAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableBooleanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableEnumAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableGuidAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\NullableSimpleTimeSpanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\ObjectAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\ReferenceTypeAssertionsSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\SimpleTimeSpanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\StringAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\TimeSpanConversionExtensionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\TimeSpanExtensions.cs" />
```

Comment:
Move `TimeSpanConversionExtensionSpecs` and `TimeSpanExtensions` (and rename to `TimeSpanExtensionSpecs`) to `extensions/`?

Reply:
`TimeSpanExtensions` is actually just code supporting the tests. Was thinking of having a `Common` folder or something to keep the root clean.